packages/tests/server/websockets.test.ts

```
@@ -1690,6 +1693,9 @@ describe('auth / connectionParams', async () => {
     whoami: t.procedure.query((opts) => {
       return opts.ctx.user;
     }),
+    iterable: t.procedure.subscription(async function* () {
+      await Promise.never();
+    }),
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)